sw/ground_segment/tmtc/rotorcraft_server.ml

```diff
@@ -174,7 +174,7 @@ let log_and_parse = fun ac_name (a:Aircraft.aircraft) msg values ->
             and desired_alt = foi32value "carrot_up" /. pos_frac in
             a.desired_pos <- Aircraft.add_pos_to_nav_ref nav_ref ~z:desired_alt (desired_east, desired_north);
             a.desired_altitude <- desired_alt +. (hmsl_of_ref nav_ref a.d_hmsl);
-            a.desired_course   <- foi32value "carrot_psi" /. angle_frac
+            a.desired_course   <- fvalue "carrot_psi" /. angle_frac
```

Comment:
As mentioned in https://github.com/paparazzi/pprzlink/pull/49 I don't think it is needed to change this message...